Text/PIIAnonymization/README.md

```diff
@@ -0,0 +1,216 @@
+# Text PII Anonymization Power Skill
+
+This Power Skill uses [Presidio](https://github.com/microsoft/presidio) Analyzer and Anonymizer
+to find and remove PII entities. Even though Presidio supports several anonymization methods (hash, encrypt, redact, replace, mask), 
```

Comment:
Lets change the order of the sentances here - put this one after explaining about what presidio is

Reply:
Done. Also renamed the package :) and added better text to the indexer.